info/flow_and_pressure.md

```diff
@@ -0,0 +1,42 @@
+**Sidenote**: users who flow-profile on manual needle valve machines will often communicate by noting the water debit at a certain needle position, and indicating that position during the brew for a repeatable flow profile, while users who own a Decent DE1 will often communicate flow rate via the calculated flow before the grouphead. In a needle valve machine, the user is not "aiming" to get 4mL/s output flow when the needle is moved there. As we know, the output flow is reliant on the grind setting and will resist the input flow accordingly. Needle valve water debit then, is a reliable way to communicate flow without needing an digitized bluetooth scale.
+
+#### Conical-Valve  
+
+ On the La Marzocco GS3 MP, profiling is achieved via the movement of a conical valve which works as an adjustable OPV, shunting off excess flow at a given pressure rather than restricting the input flow to the grouphead. Thus, the paddle position will correspond to a brew pressure rather than to an unloaded water debit. For users attempting lever type profiles, this method may be more intuitive given no need to open and shut the needle valve to maintain pressure. However, other users may find flow input profiling more intuitive and thus, the choice between the two types is one of preference over performance.
+
+## Levers
+
+In a lever machine, a piston is either forced down by a spring or the arm of a human, which then forces water through the puck. Levers are unique because they can provide nearly unlimited water debit. To the system. If you wanted to build 9 bars of pressure with ultra-coarse 4:6 method grinds, you are only limited by your ability to press down very quickly or the springs ability to uncoil. Thus, communicating in water debit is not particularly useful for levers, since they truly exert a fixed brew pressure and will supply as much flow as needed in order to maintain their set pressure, meaning the water debit "setting" during the shot pull is unknown. Because a spring will always exert the same amount of pressure, they do not (commonly) have OPV's that limit the maximum brew pressure. In addition, lever machines (dipper levers in particular) are known to have the fastest headspace fill (up to 30-80mL/s depending) because preinfusion is achieved by essentially opening a small hole to a boiler pressurized at 1-1.5 bar. Spring levers in particular will automatically execute a flow profile that preinfuses at high water debit but low pressure (boiler pressure or line pressure), which then leads into a peak pressure which declines slowly as the spring uncoils. 
```

Comment:
> unlimited water debit. To the system.

Merge sentences

> meaning the water debit "setting" during the shot pull is unknown.

I would skip mentioning water debit at all here. Maybe mention that the mental model for a lever is very different, and there isn't really an equivalent "water debit" concept for levers.

> Spring levers in particular will automatically execute a flow profile that preinfuses at high water debit but low pressure (boiler pressure or line pressure), which then leads into a peak pressure which declines slowly as the spring uncoils.

Isn't this a pressure profile and not a flow profile?